script/publish-docs

```diff
@@ -14,9 +14,9 @@ npm run doc
 [ -d tmp/website ] || {
   REPO=https://github.com/probot/probot.github.io
 
-  # If GH_TOKEN environment variable is present, use that to push.
-  if [[ -n "$GH_TOKEN" ]]; then
-    REPO=${REPO/https:\/\//https://${GH_TOKEN}@}
+  # If GITHUB_TOKEN environment variable is present, use that to push.
+  if [[ -n "$GITHUB_TOKEN" ]]; then
+    REPO=${REPO/https:\/\//https://x-access-token:${GITHUB_TOKEN}@}
```

Comment:
I think this'll fail because the `GITHUB_TOKEN` provided here isn't scoped to push to the other repo 🤔 We'll need to provide a PAT instead.

Reply:
no it will work with the `x-access-token` prefix, see https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#http-based-git-access-by-an-installation

Reply:
Ah wait I see, `REPO` seems to be an environment variable that was provided by GitHub so we need to replace that, and later for the push we need to use the authenticated git URL instead of `origin`, I'll fix that up